proto/osmosis/gamm/pool-models/balancer/balancerPool.proto

```diff
@@ -102,6 +102,15 @@ message PoolAsset {
     (gogoproto.moretags) = "yaml:\"weight\"",
     (gogoproto.nullable) = false
   ];
+
+  //   Weight that is normalized s.t. the sum of all pool assets' weights
+  // equals 1. If the user provides this value, it is ignored. This should only
+  // ever be set by the state machine. This is left as a TODO for a future PR.
```

Comment:
Let's create an issue for this TODO and remove it from comments please

Reply:
Yes. I'll remove it  and make a issue